doc/developers/utilities.rst

```diff
@@ -177,6 +177,10 @@ Benchmarking
 Testing Functions
 =================
 
+- :func:`check_memory`: check that memory is a valid input. If a string is
```

Comment:
Move to line 45. It is fitting better with a small descriptive paragraph instead of an item:

```
Some estimators are able to store intermediate results using ``Memory``
instance. The func:`check_memory`: check that memory is a valid input.
If a string is given, a ``joblib.Memory`` will be created. A user-specific 
``Memory`` object having a ``cache`` method can be passed as well.
```